sdk/cosmosdb/cosmos/src/client/Item/Items.ts

```diff
@@ -373,4 +383,68 @@ export class Items {
       ref
     );
   }
+
+  public async bulk(
+    operations: Operation[],
+    options?: RequestOptions
+  ): Promise<OperationResponse[]> {
+    const {
+      resources: partitionKeyRanges,
+    } = await this.container.readPartitionKeyRanges().fetchAll();
+    const { resource: definition } = await this.container.getPartitionKeyDefinition();
+    const batches: Batch[] = partitionKeyRanges.map((keyRange: PartitionKeyRange) => {
+      return {
+        min: keyRange.minInclusive,
+        max: keyRange.maxExclusive,
+        rangeId: keyRange.id,
+        indexes: [],
+        operations: [],
+      };
+    });
+    operations
+      .map((operation) => addPKToOperation(operation, definition))
+      .forEach((operation: Operation, index: number) => {
+        const partitionProp = definition.paths[0].replace("/", "");
```

Comment:
There is an upcoming change https://github.com/Azure/azure-cosmos-dotnet-v3/pull/1658 to expose allowing multiple paths in partition keys. We can handle it subsequently here, but FYI.

Reply:
yep, I'm prepared to handle